group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupMetadataManager.java

```diff
@@ -1805,6 +1990,280 @@ private List<ShareGroupHeartbeatResponseData.TopicPartitions> fromShareGroupAssi
             .toList();
     }
 
+    /**
+     * Handles a regular heartbeat from a Streams group member.
+     * It mainly consists of five parts:
+     * 1) Created or update the member.
+     *    The group epoch is bumped if the member has been created or updated.
+     * 2) Initialized or update the topology. The group epoch is bumped if the topology
```

Comment:
```suggestion
     * 2) Initialize or update the topology. The group epoch is bumped if the topology
```
What do you think about adding a newline after the first sentence as you did in 1)?

Reply:
Done